test/extensions/common/wasm/wasm_test.cc

```diff
@@ -354,6 +354,47 @@ TEST_P(WasmCommonTest, Foreign) {
   wasm->startForTesting(std::move(context), plugin);
 }
 
+TEST_P(WasmCommonTest, WASI) {
+  if (GetParam() != "v8") {
```

Comment:
`if (GetParam() == "null")`, since we support other runtimes.

Reply:
Fixed in the next commit in this PR.